src/__tests__/parse.test.js

```diff
@@ -38,21 +38,24 @@ describe('Parse', () => {
   });
 
   it('should parse a commit message', () => {
-    const commitMessageRefactor = '1f12345b597123453031234555b6d25574ccacee refactor(file): lorem updates (#8)';
-    const commitMessageFeature = '1f12345b597123453031234555b6d25574ccacee feat(dolor): issues/20 sit enhancements';
-    const commitMessageNonCC = '1f12345b597123453031234555b6d25574ccacee Initial commit';
+    const commitMessageRefactor = '1f1x345b597123453031234555b6d25574ccacee refactor(file): lorem updates (#8)';
+    const commitMessageFeature = '1f72345b597123453031234555b6d25574ccacee feat(dolor): issues/20 sit enhancements';
+    const commitMessageNoScope = '1f12p45b597123453031234555b6dl2401ccacee fix: missing semicolon';
+    const commitMessageNonCC = '1f12s45b597123453031234555b6d25574ccacee Initial commit';
```

Comment:
required:
expand this test for some more "alternative" formats so we can monitor the output with snapshots

Reply:
We had the quick thought that we should normalize commit messages that were missing minor bits of syntax but then backtracked. 

The reason - It was leading to more than a few assumptions in an attempt to assume they forgot something instead of assuming they formatted on purpose. We're going to avoid compensating for the forgetful user